universal/components/ContentList/component.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+import React from 'react'
+import styles from './styles.css'
+
+type ContentListProps = {
+  hasTitle: boolean,
+  title: string,
+  bulletList: boolean,
+  items: Array<string>,
+  itemWidth: 'full' | 'half' | 'quarter' | 'third'
+}
+
+// Logic for renderTitle
+function renderTitle ({
+  hasTitle,
+  title
+}: { hasTitle: boolean, title: string }) {
+  if (hasTitle) {
```

Comment:
we should avoid using a boolean here along with the property. if we just check to see if the title exists, that should be sufficient.

you can remove this entire function `renderTitle` and replace it with only the following in the body of the actual component:
```javascript
return (
<div className={styles.listWrap}>
   {title && <h1>{title}</h1>}
   {renderList({bulletList, items, itemWidth})}
  </div>
)